api/python/PE/objects/signature/pyContentInfo.cpp

```diff
@@ -40,18 +41,6 @@ void create<ContentInfo>(py::module& m) {
         &ContentInfo::content_type,
         "OID of the content type. This value should match ``SPC_INDIRECT_DATA_OBJID``")
 
-    .def_property_readonly("type",
-        &ContentInfo::type)
-
-    .def_property_readonly("digest_algorithm",
-        &ContentInfo::digest_algorithm,
-        "Algorithm (OID) used to hash the file. This value should match SignerInfo.digest_algorithm and Signature.digest_algorithm")
-
-
-    .def_property_readonly("digest",
-        &ContentInfo::digest,
-        "The digest")
-
     .def_property_readonly("raw",
```

Comment:
Can we move this `raw` field into `SpcIndirectDataContent` like the three above?  It probably makes more sense to live there anyway, since that field currently gets populated with only the SpcIndirectDataContent bytes

https://github.com/lief-project/LIEF/blob/657d589fae9949ad48b6e074517299853b4a99ba/src/PE/signature/SignatureParser.cpp#L173-L192

With the ContentInfo no longer being passed back via `Signature.content_info` (now it's a `SpcIndirectDataContent` that usually gets passed back) right now it doesn't seem like it's possible to access this `raw` field if you have a `Signature` object...

Access to the raw bytes is needed when validating the Authenticode signature... I should probably add Authenticode validation logic as a LIEF test case so it's more clear how these raw fields can be used. 🤔